cmd/svcat/binding/unbind_cmd_test.go

```diff
@@ -178,9 +217,32 @@ func TestUnbindCommand(t *testing.T) {
 			if err != nil {
 				gotOutput += err.Error()
 			}
-			if !strings.Contains(gotOutput, tc.wantOutput) {
+			if !outputMatches(gotOutput, tc.wantOutput, tc.allowDiffOrder) {
 				t.Errorf("unexpected output \n\nWANT:\n%q\n\nGOT:\n%q\n", tc.wantOutput, gotOutput)
 			}
 		})
 	}
 }
+
+func outputMatches(gotOutput string, wantOutput string, allowDifferentLineOrder bool) bool {
```

Comment:
🤗 This is just what these tests needed!

Reply:
Yeah, it's not perfect (because the first line with --wait should always be "waiting for the binding(s) to be deleted..."), but I feel handling this would complicate things too much.